src/Components/src/Microsoft.AspNetCore.Components.Build/targets/RazorCompilation.targets

```diff
@@ -12,6 +12,7 @@
     <GenerateRazorAssemblyInfo>false</GenerateRazorAssemblyInfo>
     <GenerateRazorHostingAssemblyInfo>false</GenerateRazorHostingAssemblyInfo>
     <GenerateProvideApplicationPartFactoryAttribute>false</GenerateProvideApplicationPartFactoryAttribute>
+    <_RazorComponentInclude>**\*.cshtml</_RazorComponentInclude>
```

Comment:
Which component uses this property?

Reply:
Not sure. I was just merging in https://github.com/aspnet/AspNetCore/pull/6713 which included the razor SDK update